frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -70,6 +80,24 @@ const TeamEventCreditForm: React.FC = () => {
         contentMsg: 'Team events must be logged for at least 0.5 hours!'
       });
     } else {
+
+      images.map(async (image, index) => {
+        const newTeamEventAttendance: TeamEventAttendance = {
+          member: userInfo,
+          hoursAttended: teamEvent.hasHours ? Number(hours) : undefined,
+          image: `eventProofs/${getNetIDFromEmail(userInfo.email)}/${new Date().toISOString()}`,
+          eventUuid: teamEvent.uuid,
+          pending: true,
+          status: 'pending' as Status,
+          reason: '',
+          uuid: ''
+        };
+
+        const createdAttendance = await requestTeamEventCredit(
+          newTeamEventAttendance,
+          images[index]
+        );
+
       const newTeamEventAttendance: TeamEventAttendance = {
```

Comment:
Do we need both `newTeamEventAttendance` and `newTeamEventCreditAttendance` here?

Reply:
@andrew032011 the check that did not pass said `cannot redeclare block-scoped variable 'newteameventattendance'`.

Reply:
Right, but those two variables look almost the exact same, do we need both?

Reply:
Ah, I see what you meant! I removed one of them as I was cleaning up code